src/main/java/de/dhbw/mh/slang/craft/parser/CraftedSlangParser.java

```diff
@@ -363,26 +364,52 @@ public AstNode exponent2( AstNode previous ){
 	
 	@Override
 	public AstNode atomicExpression( ){
-		// TODO Auto-generated method stub
-		return super.atomicExpression( );
+		switch (this.LEXER.lookahead().TYPE) {
+			case LPAREN:
+				return this.atomicExpression1();
+			case IDENTIFIER:
+				return this.atomicExpression2();
+			case NUMERIC_LITERAL:
+			case TRUE:
+			case FALSE:
+				return this.atomicExpression3();
+			default:
+				throw parsingException(Selector.ATOMICS);
+		}
 	}
 	
 	@Override
 	public AstNode atomicExpression1( ){
-		// TODO Auto-generated method stub
-		return super.atomicExpression1( );
+		match(LPAREN);
+		AstNode node = this.conditionalExpression();
+		match(RPAREN);
+
+		return node;
 	}
 	
 	@Override
 	public AstNode atomicExpression2( ){
-		// TODO Auto-generated method stub
-		return super.atomicExpression2( );
+		if (this.LEXER.lookahead().TYPE != IDENTIFIER) {
+			throw this.parsingException(IDENTIFIER);
+		}
+
+		AstVariable variable = new AstVariable(this.LEXER.lookahead().LEXEME);
+
+		this.LEXER.advance();
+
+		return variable;
```

Comment:
Why are there so many blank lines?

Reply:
well.... good point :D

Feel free to remove them in a follow-up. :)